partcad-cli/src/partcad_cli/click/command.py

```
@@ -49,7 +50,7 @@
     show_envvar=True,
 )
 @click.pass_context
-def cli(ctx, v, q, no_ansi, p, format):
+def cli(ctx, verbose, quiet, no_ansi, package, format):
```

Ruff (0.8.2)

Missing return type annotation for public function `cli`

Add return type annotation: `None`

(ANN201)